### DISCUSSION TITLE
What's the use giving student pack perks if it makes people struggle?

### DISCUSSION THREAD

**Discussion by nndmk** [community/community#134963] [GitHub Education]
### Hi everyone,

Trying to apply for GitHub education pack is waste of time. I thought it would gonna be like how Spotify offers discount for student, verifying our credibility using SheerID, and then done leave it to SheerID to verify. But in GitHub? I tried using desktop browser to upload my student ID, and then it takes me to open my webcam? You can ask my campus if you don't believe me, I don't mind giving you the details of where I go study. We don't have a printed student ID card, we only can access our ID online using our student portal, which can be downloaded as jpg / pdf. Same with our transcript etc. How in the world I'm applying it using webcam? I'm using ASUS laptop, come on what can you expect from this webcam idea. It's all blurred. I tried to be patient and try safari mobile browser from my iOS, but it only works using front camera? There's no way I could access my back camera, I was so pathetic using my front camera to shoot on my laptop screen that was showing the student ID from student portal.

If you really wanna give this pack to students, try to make their life easier, improve how you're receiving their request. Other company like Spotify and others use a much advanced system than this to verify student's credibility, Notion offers student perks only by registering using school email, and I'm using their apps. I love Notion for how easy it is to use and it's so simple, but I just wanna try to be a bit more flexible and being creative here????? that's what brought me to your website. Why did you still using this kind of frustrating way? If you're really not interested in giving this perks for student or not interested to make people's life easier, then just cancel this perks program. The reason I wanna use GitHub is because in this new academic year, I'm officially a final year student, and I just wanna build a simple portfolio in order to build my career from scratch, sharing my ideas and profiles for jobhunting process and be creative by using GitHub. I was so desperate looking for ways to contact any customer support available in this platform, just to be able to say I can't do what you required from me, and even uploaded all my students document if it's needed. Instead, I'm brought here, I don't know if someone from GitHub will ever read it or not. Omg.

**Comment by abatker** [comment]
I am having issues with uploading my information as well! My laptop cam is technically HD, and video calls, including ones for projects I do for classes, work perfectly fine. I don't see why we cant post a screenshot that is much more clear that demonstrates I attend my university! I am a final year student as well. There is no email for support, so I was brought here as well.... Hopefully someone reads this too!

**Comment by rohanvkumarv** [comment]
fr fr this was a headache for me too fr ....i mean why not just add a screenshot...i had to use my mobile as webcam lol to finally do something
